File: tests/fixtures/typescript/simple.ts
import { join } from "./utils";
import { readFileSync } from "node:fs";

export const MAX_RETRIES = 3;

export function greet(name: string): string {
  return `Hello, ${name}!`;
}

export function add(a: number, b: number): number {
  return a + b;
}

function internalHelper(): void {
  console.log("not exported");
}

File: tests/fixtures/go/simple.go
package main

import (
	"fmt"
	"errors"
)

func Greet(name string) string {
	return fmt.Sprintf("Hello, %s!", name)
}

func Add(a, b int) int {
	return a + b
}

func Divide(a, b float64) (float64, error) {
	if b == 0 {
		return 0, errors.New("division by zero")
	}
	return a / b, nil
}

func internalHelper() {
	fmt.Println("not exported")
}
